src/cmds/qsub.c

```diff
@@ -4543,17 +4550,18 @@ print_usage()
 	static char usage[]=
 		"usage: qsub [-a date_time] [-A account_string] [-c interval]\n"
 	"\t[-C directive_prefix] [-e path] [-f ] [-G] [-h ] [-j oe|eo] [-J X-Y[:Z]]\n"
-	"\t[-k o|e|oe] [-l resource_list] [-m mail_options] [-M user_list]\n"
+	"\t[-k (d|o|e|n)*] [-l resource_list] [-m mail_options] [-M user_list]\n"
```

Comment:
I believe the common method of saying a required argument in usage is by putting the argument in \<\> instead of using regular expressions like * or +.